latest/mkdocs.yml

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Do we want to maintain this copy? Can't we do a `cp` in the CI? WDYT?

Reply:
good point, I was going to have it in the CI anyways 😂.